README.md

```diff
@@ -17,31 +16,43 @@ See our [LTS policy](https://github.com/vtex-apps/awesome-io#lts-policy) for mor
 
 ## Table of Contents
 - [Usage](#usage)
-- [API](#api)
   - [Blocks API](#blocks-api)
     - [Configuration](#configuration)
   - [Styles API](#styles-api)
 - [Troubleshooting](#troubleshooting)
 
 ## Usage
-:construction: :construction: :construction:
 
-To use this app you need to import in your dependencies on `manifest.json`.
+This app uses our store builder with the blocks architecture. To know more about Store Builder [click here.](https://help.vtex.com/en/tutorial/understanding-storebuilder-and-stylesbuilder#structuring-and-configuring-our-store-with-object-object)
+
+To use this app, you need to import in your dependencies on `manifest.json`.
 
 ```json
   dependencies: {
     "vtex.minicart": "2.x"
   }
 ```
 
-## API 
-:construction: :construction: :construction:
+Then, add `minicart` block into our app theme, as we do in our [Dreamstore app](https://github.com/vtex-apps/dreamstore/blob/master/store/blocks.json). 
 
 ### Blocks API
 :construction: :construction: :construction:
 
+This app has an interface that describes what the rules that must a block implement when you want to use the minicart. 
```

Comment:
I think that this text would be better this way:

```
This app has an interface that describes what the rules that must be implemented by a block when you want to use the minicart. 
```

Reply:
`This app has an interface that describes what rules must be implemented by a block when you want to use the minicart. `

What do you think?

Reply:
I think it's great! :100:

Reply:
which rules*